cardano-node/src/Cardano/Node/Parsers.hs

```diff
@@ -100,24 +100,34 @@ parseHostIPv4Addr =
     option (eitherReader parseNodeHostIPv4Address) (
           long "host-addr"
        <> metavar "IPV4"
-       <> help "An optional ipv4 address"
+       <> help "An optional IPv4 address"
     )
 
 parseHostIPv6Addr :: Parser NodeHostIPv6Address
 parseHostIPv6Addr =
     option (eitherReader parseNodeHostIPv6Address) (
           long "host-ipv6-addr"
        <> metavar "IPV6"
-       <> help "An optional ipv6 address"
+       <> help "An optional IPv6 address"
     )
 
 parseNodeHostIPv4Address :: String -> Either String NodeHostIPv4Address
 parseNodeHostIPv4Address str =
-   maybe (Left $ "Failed to parse: " ++ str) (Right . NodeHostIPv4Address) $ readMaybe str
+  maybe
+    (Left $
+      "Failed to parse: " ++ str ++
+      ". Use --host-ipv6-addr to specify IPv6 adddress")
+    (Right . NodeHostIPv4Address)
+    (readMaybe str)
 
 parseNodeHostIPv6Address :: String -> Either String NodeHostIPv6Address
 parseNodeHostIPv6Address str =
-   maybe (Left $ "Failed to parse: " ++ str) (Right . NodeHostIPv6Address) $ readMaybe str
+  maybe
+    (Left $
+      "Failed to parse: " ++ str ++
```

Comment:
Does this solve the issue the user was having? I.e does the IPv6 address "2601:640:8880:375b:2c0:caff:fe47:9d63" successfully parse now?

Reply:
Also I think you mixed up your error messages.

Reply:
```
$ cardano-node run --host-addr 2601:640:8880:375b:2c0:caff:fe47:9d63
option --host-addr: Failed to parse: 2601:640:8880:375b:2c0:caff:fe47:9d63. Use --host-ipv6-addr to specify IPv6 adddress
```

Reply:
I don't see anything mixed up. V4 error suggests V6 and vice versa.